apps/web/src/data/de/math-exams-taxonomies.ts

```diff
@@ -9,3 +9,41 @@ export const mathExamsTaxonomies = [...mathExamTaxNI, ...mathExamTaxBY]
 export const schoolTaxonomies = [
   201593, 16259, 16157, 16042, 97943, 97944, 97945, 97946, 97947, 16376, 16033,
 ]
+
+export const extraMetaTags = {
+  297606: {
+    title: 'Abiturprüfungen Mathe eA mit Lösungen',
+    metaDescription:
+      'Mathe lernen fürs Abitur mit Serlo. Mathe Abi Aufgaben erhöhtes Anforderungsnvieau mit Lösungen. Du schaffst das!',
+  },
+  300778: {
+    title: 'Abiturprüfungen Mathe gA mit Lösungen',
+    metaDescription:
+      'Mathe lernen fürs Abitur mit Serlo. Mathe Abi Aufgaben grundlegendes Anforderungsnvieau mit Lösungen. Du schaffst das!',
+  },
+  297604: {
+    title: 'Mathe Prüfungsaufgaben mit Lösungen | Realschule',
+    metaDescription:
+      'Mathe lernen für die Realschul- Abschlussprüfungen mit Original- Aufgaben und Lösungshilfe. Mit Serlo schaffst du das!',
+  },
+  302433: {
+    title: 'IGS-Prüfungen Mathe G-Kurs mit Lösungen',
+    metaDescription:
+      'Mathe lernen mit Original Aufgaben und Lösungen für die Abschlussprüfungen an der Gesamtschule. Mit Serlo schaffst du das!',
+  },
+  20852: {
+    title: 'Mathe Abiturprüfungen mit Lösung',
+    metaDescription:
+      'Die beste Vorbereitung fürs Mathe Abi. Mathe-Aufgaben fürs Abitur mit Lösungen und Erklärungen. Mit Serlo schaffst du das!',
+  },
+  75049: {
+    title: 'Mathe Abschlussprüfungen mit Lösung | Mittlere Reife',
+    metaDescription:
+      'Deine Vorbereitung für die Mittlere Reife Zweig i: Mathe lernen mit Original Prüfungsaufgaben. Mit Serlo schaffst du das!',
+  },
+  76750: {
+    title: 'Mathe Abschlussprüfungen mit Lösung | Mittlere Reife',
+    metaDescription:
+      '	Deine Vorbereitung für die Mittlere Reife Zweig ii und iii: Mathe lernen mit Original Prüfungsaufgaben. Mit Serlo schaffst du das!',
+  },
+} as const
```

Comment:
Curious why `as const` is necessary, but not a blocker

Reply:
might not strictly be necessary in this case tbh but it's just so nice in many cases so I just add it most of the times. 
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions